src/core/public/application/types.ts

```diff
@@ -114,6 +114,14 @@ export interface AppMountContext {
     overlays: OverlayStart;
     /** {@link UiSettingsClient} */
     uiSettings: UiSettingsClientContract;
+    /**
+     * exposed temporarily until https://github.com/elastic/kibana/issues/41990 done
+     * use *only* to retrieve config values
+     * @deprecated
+     * */
```

Comment:
It might be worth noting that NP plugins have no way of exposing injected vars from the server-side. It must be done from a legacy plugin.